client/src/utils/styles.scss

```
@@ -0,0 +1,23 @@
+/* stylelint-disable at-rule-no-unknown */
+
+$min-content: strip(theme('screens.screen-300'));
+
+/**
+ * Function for clamping a min and max value scaled based on the ratio between
+ * the min value and the min content width to ensure proper scaling.
+ */
+@function viewport-clamp($min-value, $max-value) {
+  $value: clamp(
+    #{$min-value},
+    calc(strip(#{$min-value}) / #{$min-content} * 100vw),
+    #{$max-value}
+  );
+
+  @return $value;
+}
+/**
+ * Function for calculating the rem value for another value.
+ */
+@function calculate-rem($value) {
+  @return $value / 16px * 1rem;
```

nit: could you make `16px` a constant value with a name denoting that it's the default font size

also I think it might be slightly cleaner to write this as

```suggestion
  @return ($value / 16px) rem;
```

---

also the docstring should specify that this is for calculating pixels as rem, not any unit type

---

But PEMDAS 😢  /s

Sounds good, I'll make a tiny PR to update this 😆